addons/mail/static/src/models/discuss/discuss.js

```diff
@@ -272,6 +272,23 @@ function factory(dependencies) {
             return [];
         }
 
+        /**
+         * @private
+         */
+        _onChangeThreadIsPinned() {
+            let thread = this.thread;
+            // No thread, or thread is being removed
+            // So we display discuss' default thread (usually the Inbox)
+            if (!thread || !thread.isPinned) {
+                const [model, id] = this.defaultInitActiveId.split('_');
+                thread = this.env.models['mail.thread'].find(thread =>
+                    thread.id === id && thread.model === model
+                );
```

Comment:
Was this `defaultInitActiveId` used anywhere before this PR?

Can't we store a thread relation to inbox instead of a string that we parse?